examples/formats/json/python/func.yaml

```diff
@@ -0,0 +1,7 @@
+name: denismakogon/hot-json-python:0.0.1
+version: 0.0.1
+runtime: docker
```

Comment:
Why not just python?

Reply:
`python3.5.2` or `python3.6.2` to be more precise.